src/dotnet-compile-fsc/Program.cs

```diff
@@ -295,5 +278,100 @@ private static IEnumerable<string> GetSourceFiles(IReadOnlyList<string> sourceFi
 
             yield return sourceFiles.Last();
         }
+
+        private static Command RunFsc(List<string> fscArgs, string temp)
+        {
+            var fscEnvExe = Environment.GetEnvironmentVariable("DOTNET_FSC_PATH");
+            var exec = Environment.GetEnvironmentVariable("DOTNET_FSC_EXEC")?.ToUpper() ?? "COREHOST";
+            
+            var muxer = new Muxer();
+
+            if (fscEnvExe != null)
+            {
+                switch (exec)
+                {
+                    case "RUN":
+                        return Command.Create(fscEnvExe, fscArgs.ToArray());
+
+                    case "COREHOST":
+                    default:
+                        var host = muxer.MuxerPath;
+                        return Command.Create(host, new[] { fscEnvExe }.Concat(fscArgs).ToArray());
+                }
+            }
+            else
+            {
+                var fscCommandSpec =  ResolveFsc(fscArgs, temp)?.Spec;
+                return Command.Create(fscCommandSpec);
```

Comment:
Will look into this, I had it originally. The specific logic we need to get dotnet-compile-fsc working until packaging changes are made for fsc may block this.